R/drive_mkdir.R

```diff
@@ -54,9 +55,18 @@ drive_mkdir <- function(path = NULL, name = NULL, verbose = TRUE) {
 
   path <- path %||% root_folder()
   path <- as_dribble(path)
-  confirm_single_file(path)
+  if (!single_file(path)) {
```

Comment:
For some reason to me (at the moment) `drive_mkdir()` it makes sense to default to making a directory in your root directory, but for `drive_mv()` if you don't give a good place for it to move to, I'm not sure that the root directory makes the most sense? And for `drive_cp()`, I think maybe the default should actually be sticking it in the folder the original file is in rather than ow we have it error now. What do you think?